### PR TITLE
cli: workflow: show: print env vars

### PR DESCRIPTION
**Pull request by danielnorberg**
# Hey, I just made a Pull Request!

## Checklist for PR author(s)

- [x] Changes are covered by unit test
- [x] All tests pass
- [x] Code coverage check passes
- [ ] Error handling is tested
- [ ] Errors are handled at the appropriate layer
- [ ] Errors that cannot be handled where they occur are propagated
- [ ] (optional) Changes are covered by system test
- [ ] Relevant documentation updated
- [ ] This PR has NO breaking change to public API
- [x] This PR has breaking change to public API and it is documented

## Checklist for PR reviewer(s)

- [x] This PR has been incorporated in release note for the coming version
- [x] Risky changes introduced by this PR have been all considered

### CODE REVIEW

**Comment by honnix** [comment]
👍

File: styx-cli/src/main/java/com/spotify/styx/cli/PlainCliOutput.java [hunk 1]
```diff
@@ -120,7 +120,7 @@ public void printMessage(String message) {
   public void printWorkflow(Workflow wf, WorkflowState state) {
     System.out.println(Joiner.on(' ').join(
         wf.componentId(),
-        wf.id(),
+        wf.id().id(),
```
Comment (honnix) [inline styx-cli/src/main/java/com/spotify/styx/cli/PlainCliOutput.java]: This fix breaks backward compatibility a bit.

**Comment by honnix** [comment]
No breaking change to API but command line output changes a bit.

File: styx-cli/src/main/java/com/spotify/styx/cli/PlainCliOutput.java (hunk 1 above)
Comment (danielnorberg) [reply styx-cli/src/main/java/com/spotify/styx/cli/PlainCliOutput.java]: I'll be very surprised if anyone was relying on this

Comment (honnix) [reply styx-cli/src/main/java/com/spotify/styx/cli/PlainCliOutput.java]: Yeah, I didn't find anything.

**Pull request merged**